### PR TITLE
Bring back recommended practices for writing roles

### PR DESCRIPTION
**Pull request by MiLk**
Even if roles are not in AerisCloud, I think this document should be present and it describes what is enforced by the rules provided with AerisCloud.

cc @stelcheck

### CODE REVIEW

**Comment by stelcheck** [comment]
Well since we have a whole organization, how about we create a separate repos for documenting the guidelines for creating roles, alongside perhaps a process through which third-parties could submit new roles and join the organization?

**Comment by stelcheck** [comment]
Other than that, LGTM

**Comment by MiLk** [comment]
There is work in progress at several places:
- https://github.com/willthames/ansible-lint/pull/137
- https://github.com/ansible/galaxy-issues/issues/27
- https://github.com/ansible/galaxy-issues/issues/31

I want to see how it will evolve since ansible 2 has been released, galaxy is going forward and maybe ansible-lint v3.

If nothing is happening, we can start something and get people involved.

File: ansible/rules/README.md [hunk 1]
```diff
@@ -6,22 +6,93 @@ This folder contains the rules used by [ansible-lint](https://github.com/willtha
...
+
+#### Structure
+
+All tasks **must**:
+
+* Have a descriptive name, which explains what the task will be doing
+* When using variables, use spaces between the variable name and
+  the opening/closing brackets
+* Actions with arguments must be entered in multiline mode
+* Use shell for any command execution, and use multiline (However the shell module should be avoided as much as possible).
```
Comment (christopherobin) [inline ansible/rules/README.md:46]: Also tags are mandatory

**Comment by christopherobin** [comment]
:+1:

**Pull request merged**